docker-compose/monitor/README.md

```diff
@@ -0,0 +1,173 @@
+# atm-dev
+Development environment for ATM
```

Comment:
what is ATM? Please spell out.

Reply:
Thanks for picking up on this, I've elaborated a little more on what ATM stands for, the architecture of the development/demo environment and explanation of each component with relevant links, and an illustration of how the components interact with one another.